ArmoniK.SDK.Client.Test/CMakeLists.txt

```diff
@@ -1,31 +1,63 @@
 cmake_minimum_required(VERSION 3.22)
 set(PROJECT_NAME ArmoniK.SDK.Client.Test)
 set(NAMESPACE ArmoniK::Sdk::Client::Test)
+set(version 0.1.0)
+
 project(${PROJECT_NAME})
+set(CMAKE_INSTALL_INCLUDEDIR ${CMAKE_INSTALL_PREFIX}/include)
```

Comment:
I don't think it's needed

Reply:
It's not needed but it simplifies the build when performing in two steps. I removed it